source/api/AST/ASTVisitor.cpp

```diff
@@ -1292,10 +1294,17 @@ ASTVisitor::
 WalkUpFromParmVarDecl(
     ParmVarDecl* D)
 {
-    // We do nothing here, to prevent ParmVarDecl
-    // from appearing as VarDecl. We pick up the
-    // function parameters as a group from the
-    // FunctionDecl instead of visiting ParmVarDecl.
+    Assert(shouldTraversePostOrder() &&
+        "expected post-order traversal");
```

Comment:
oh man I hate this "text" in the assert

Reply:
How about "modifying a ParamVarDecl here depends on post-order traversal"?

Reply:
I dont want any text at all, it is gratuituous because the assert will probably never  go off so we are paying for a string for nothing. I edited the code already to remove the text and just leave a comment.